docker/test/stateless/run.sh

```diff
@@ -85,6 +85,45 @@ fi
 
 sleep 5
 
+# Set follow-fork-mode to parent, because we attach to clickhouse-server, not to watchdog
+# and clickhouse-server can do fork-exec, for example, to run some bridge.
+# Do not set nostop noprint for all signals, because some it may cause gdb to hang,
+# explicitly ignore non-fatal signals that are used by server.
+# Number of SIGRTMIN can be determined only in runtime.
+RTMIN=$(kill -l SIGRTMIN)
+echo "
+set follow-fork-mode parent
+handle SIGHUP nostop noprint pass
+handle SIGINT nostop noprint pass
+handle SIGQUIT nostop noprint pass
+handle SIGPIPE nostop noprint pass
+handle SIGTERM nostop noprint pass
+handle SIGUSR1 nostop noprint pass
+handle SIGUSR2 nostop noprint pass
+handle SIG$RTMIN nostop noprint pass
+info signals
+continue
+backtrace full
+thread apply all backtrace full
+info registers
+disassemble /s
+up
+disassemble /s
+up
+disassemble /s
+p \"done\"
+detach
+quit
+" > script.gdb
+
+# FIXME Hung check may work incorrectly because of attached gdb
+# 1. False positives are possible
+# 2. We cannot attach another gdb to get stacktraces if some queries hung
```

Comment:
p. 1 is outdated

Reply:
Ok, I will update comment. And I can move it to some library to avoid code duplication